File: IR LLVM/Micro05/Micro05.c
#include <stdio.h>
#include <stdlib.h>

int main(){
    char nome[45];
    char sexo;
    int x;
    int m;
    int h;
    int max;

    max = 5;
    h = 0;
    m = 0;

    for(x=1; x <= max; x++){
        printf("Digite o nome: ");
        scanf("%s", nome);
        printf("Digite H  para Homem ou M para Mulher: ");
        scanf("%s", &sexo);
        switch(sexo){
            case 'H':
                h++;
                break;
            case 'M':
                m++;
                break;
            default:
                printf("Sexo só pode ser H ou M");
                break;
        }
    }
    printf("Foram inseridos %d Homens\n", h);
    printf("Foram inseridos %d Mulheres\n", m);
    return 0;
}

File: PASCAL/Micro05.pas
program micro05;
var 
    nome: string[45];
    sexo: char;
    x, h, m, max: integer;
begin
    max := 5;
    h := 0;
    m := 0;
    for x := 1 to max do
    begin
        write('Digite o nome: ');
        readln(nome);
        write('Digite H - para Homem ou M- para Mulher: ');
        readln(sexo);
        if(sexo = 'H') then
            h := h + 1;
        if  (sexo = 'M') then
            m := m + 1;
        if (sexo <> 'M') AND (sexo <> 'H') then
            writeln('Sexo so pode ser H ou M');
    end;
    write('Foram inseridos, ');
    write(h);
    writeln(' Homens');
    write('Foram inseridos ');
    write(m);
    writeln(' Mulheres');
end.